addons/website_event_track/views/event_track_views.xml

```diff
@@ -161,7 +164,7 @@
                         <div class="flex-grow-1">
                             <label for="name"/>
                             <h1>
-                                <field name="name" placeholder="e.g. Inspiring Business Talk"/>
+                                <field name="name" placeholder="e.g. Inspiring Business Talk" class="w-75"/>
```

Comment:
This actively makes it smaller for small resolutions.
Would try "w-sm-75" instead.